examples/README.md

```diff
@@ -0,0 +1,60 @@
+# examples
+
+This directory contains example code for both installing Contour and its associated Envoy Pods under various conditions, and also for setting up various extensions and other pieces of software that may help with running Contour.
+
+## Directories
+
+Most of the directories contain a complete set of Kubernetes YAML that can be applied to a cluster. This section describes the purpose of each directory, and inside each directory is a README.md that describes the details.
+
+### [`contour`](./contour/README.md)
+
+This is the recommended example installation of Contour, and will deploy Contour into a Deployment, and Envoy into a Daemonset, with the gRPC communication secured with certificates, and a `LoadBalancer` Service to expose Envoy to your upstream load balancer provider.
```

Comment:
top tip: instead of writing `, and` Close the sentence and open a new one.
```
This is the recommended example installation of Contour.
It will deploy Contour as a Deployment, and Envoy as a Daemonset.
```